elastic/kibana
# [Snapshot & Restore] Migrate away from _cat/plugins

Fixes #57287
## Summary
This PR changes the server code to not use `_cat/plugins` API and instead use `_nodes/plugins` API. I also removed a misleading comment about permissions because the API request is done as the current user. 

### How to test
1. Download 8.0.0-beta1 ES snapshot [here](https://www.elastic.co/downloads/elasticsearch#preview-release) 
2. In the ES folder, install 4 repository plugins that are checked in the code 
- azure `sudo bin/elasticsearch-plugin install repository-azure`
- s3 `sudo bin/elasticsearch-plugin install repository-s3`
- hdfs `sudo bin/elasticsearch-plugin install repository-hdfs`
- gcs `sudo bin/elasticsearch-plugin install repository-gcs`
4. In the ES folder, update `config/elasticsearch.yml` file to read `xpack.security.enabled: false`. This will disable security and no login will be asked when opening Kibana.
5. In the ES folder, start ES with `bin/elasticsearch`
3. In the Kibana folder, add to the `config/kibana.dev.yml` file 
```
elasticsearch.ignoreVersionMismatch: true
```
4. Start kibana with `yarn start`
5. Navigate to "Snapshot and Restore" and click button "Register a repository" 
7. Check that all 4 installed plugins are listed. 


### Checklist
